mlir/include/mlir/Dialect/Rock/Generator/ConvGenerator.h

```diff
@@ -24,8 +24,10 @@ namespace mlir {
 namespace rock {
 struct ConvolutionDims;
 
-class Conv2dGenerator {
+class ConvGenerator {
 public:
+  enum DIM { HEIGHT = 0, WIDTH = 1 };
```

Comment:
Symbolic constants for dimensions within the vector.  Probably should have DEPTH, too, since it's common.